articles/active-directory/external-identities/b2b-quickstart-invite-powershell.md

```diff
@@ -35,33 +35,32 @@ You need a test email account that you can send the invitation to. The account m
 Run the following command to connect to the tenant domain:
 
 ```powershell
-Connect-AzureAD -TenantDomain "<Tenant_Domain_Name>"
+Connect-MgGraph -Scopes user.readwrite.all
 ```
-For example, `Connect-AzureAD -TenantDomain "contoso.onmicrosoft.com"`.
 
 When prompted, enter your credentials.
 
 ## Send an invitation
 
-1. To send an invitation to your test email account, run the following PowerShell command (replace **"Sanda"** and **sanda\@fabrikam.com** with your test email account name and email address): 
+1. To send an invitation to your test email account, run the following PowerShell command (replace **"John Doe"** and **john\@contoso.com** with your test email account name and email address): 
 
    ```powershell
-   New-AzureADMSInvitation -InvitedUserDisplayName "Sanda" -InvitedUserEmailAddress sanda@fabrikam.com -InviteRedirectURL https://myapps.microsoft.com -SendInvitationMessage $true
+   New-MgInvitation -InvitedUserDisplayName "John Doe" -InvitedUserEmailAddress John@contoso.com -InviteRedirectUrl "https://myapplications.microsoft.com" -SendInvitationMessage:$true
    ```
 2. The command sends an invitation to the email address specified. Check the output, which should look similar to the following:
 
-   ![PowerShell output showing pending user acceptance](media/quickstart-invite-powershell/powershell-azureadmsinvitation-result.png)
+   ![image](https://user-images.githubusercontent.com/49490355/152628050-277376df-cc1f-42c6-8a81-a2dbba66b8d8.png)
```

Comment:
Can you include the new output screenshot along with this PR in the media/quickstart-invite-powershell folder? You can use the filename "powershell-mginvitation-result.png" (all lowercase), and update line 52 to reference it:

```suggestion
   ![PowerShell output of the invitation command](media/quickstart-invite-powershell/powershell-mginvitation-result.png)
```